src/nomad_simulations/schema_packages/physical_property.py

```diff
@@ -146,109 +81,113 @@ class PhysicalProperty(ArchiveSection):
         """,
     )
 
+    physical_property_ref = Quantity(
+        type=Reference(SectionProxy('PhysicalProperty')),
+        description="""
+        Reference to the `PhysicalProperty` section from which the physical property was derived. If `physical_property_ref`
+        is populated, the quantity `is_derived` is set to True via normalization.
+        """,
+    )
+
     is_scf_converged = Quantity(
         type=bool,
         description="""
         Flag indicating whether the physical property is converged or not after a SCF process. This quantity is connected
         with `SelfConsistency` defined in the `numerical_settings.py` module.
         """,
-    )
+    )  # ? tie to calculation, not individual property
 
     self_consistency_ref = Quantity(
         type=SelfConsistency,
         description="""
         Reference to the `SelfConsistency` section that defines the numerical settings to converge the
         physical property (see numerical_settings.py).
         """,
+    )  # ? remove
+
+    contributions = SubSection(
```

Comment:
I think the reason we did not include contributions in PP before was 
1. To prevent deeper nested structures of sub-classifications for a property -- Can we restrict the usage of this to be 1-level deep?
2. We were a little afraid of enabling a mess of non-standardized or redundant contributions for some property sub-class. I think we have moved beyond this perspective now in favor of more flexibility, but just to clarify

Reply:
Regarding your point 2, the presence of a quantity or subsection does not require that it gets populated.

Reply:
For point 1, there's a trade-off. Before, you had to define a new `Contribution` section for each child instance of `PhysicalProperty`.
Working from the assumption that most contributions **may** contain similar (meta)data as their containing property section, we'd likely prefer to mirror it by default. This, of course, means that you also mirror `contributions` again. 

I consulted also with Theodore, and we agree that the schema-route for preventing nesting isn't the way to go. While semantically clearer, it would entail significant effort to provide this new _exclusion_ feature.
Instead, we suggest to use normalization for raising an "error" when `contributions` is nested.

Reply:
Btw, `type` or `label` could be repurposed to distinguish contributions. For now, I'll just introduce a new quantity.

Reply:
I've added the check at the normalization level, though in principle this should also work for `__setattr__`.
I held off on this approach, as I'm unsure about the overhead it'd create.